<!-- MHonArc v2.6.16 -->
<!--X-Subject: Re: [bug #11454] Constraint incorrectly supplied when using the	"full_analysis.py" script -->
<!--X-From-R13: "Sqjneq q'Ohiretar" <rqjneq.qnhiretarNtznvy.pbz> -->
<!--X-Date: Tue, 08 Apr 2008 09:30:16 +0200 -->
<!--X-Message-Id: 7f080ed10804080030n7c8be916ofed19717f05f8a7d@mail.gmail.com -->
<!--X-Content-Type: text/plain -->
<!--X-Reference: 20080407&#45;165459.sv7147.46735@gna.org -->
<!--X-Reference: 20080407&#45;170848.sv7147.92162@gna.org -->
<!--X-Reference: 20080408&#45;005357.sv0.79865@gna.org -->
<!--X-Reference: 20080407&#45;220535.sv7147.14029@gna.org -->
<!--X-Head-End-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
   "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Re: [bug #11454] Constraint incorrectly supplied when using the	&quot;full_analysis.py&quot; script -- April 08, 2008 - 09:30</title>
<link rel="stylesheet" type="text/css" href="/mail.gna.org/archives-color-gna.css"> 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
</head>
<body>
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<h2><img src="/mail.gna.org/images/mail.orig.png" width="48" height="48"
alt="mail" class="pageicon" />Re: [bug #11454] Constraint incorrectly supplied when using the	&quot;full_analysis.py&quot; script</h2>
<br />
<div class="topmenu">
<a href="../" class="tabs">Others Months</a> | <a href="index.html#00042" class="tabs">Index by Date</a> | <a href="threads.html#00042" class="tabs">Thread Index</a><br />
<span class="smaller">&gt;&gt;&nbsp;&nbsp;
[<a href="msg00041.html">Date Prev</a>] [<a href="msg00043.html">Date Next</a>] [<a href="msg00041.html">Thread Prev</a>] [<a href="msg00043.html">Thread Next</a>]
</div>

<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<h3><a name="header" href="#header">Header</a></h3>
<!--X-Subject-Header-End-->
<!--X-Head-of-Message-->
<ul class="headdata">
<li class="menuitem">
<em>To</em>: &quot;Sébastien Morin&quot; &lt;NO-REPLY.INVALID-ADDRESS@xxxxxxx&gt;</li>
<li class="menuitem">
<em>Date</em>: Tue, 8 Apr 2008 09:30:06 +0200</li>
<li class="menuitem">
<em>Cc</em>: relax-devel@xxxxxxx</li>
<li class="menuitem">
<em>Message-id</em>: &lt;<a href="msg00042.html">7f080ed10804080030n7c8be916ofed19717f05f8a7d@mail.gmail.com</a>&gt;</li>
<li class="menuitem">
<em>References</em>: &lt;20080407-165459.sv7147.46735@xxxxxxx&gt;	&lt;20080407-170848.sv7147.92162@xxxxxxx&gt;	&lt;20080408-005357.sv0.79865@xxxxxxx&gt;	&lt;20080407-220535.sv7147.14029@xxxxxxx&gt;</li>
</ul>
<!--X-Head-of-Message-End-->
<!--X-Head-Body-Sep-Begin-->
</div><!-- end headdata -->
<br />
<h3><a name="content" href="#content">Content</a></h3>
<div class="postedby">Posted by <strong>Edward d'Auvergne</strong> on April 08, 2008 - 09:30:</div>
<div class="msgdata">
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<pre style="margin: 0em;">Hi,

For model m0, this shouldn't be a problem.  This model has no
parameters, hence the dimension of A is 0x0 and for b is 0 (these are
the constraint matrices such that A.x &lt;= b, where x is the parameter
vector).  Therefore it doesn't mater if minimisation for the
individual model-free model fails, because there is nothing to
minimise.  Code could be added to catch this model and skip it, but
I'd prefer that this behaviour stays.  Adding new code for model m0
could introduce bugs, whereas the current behaviour means nothing
happens for this model.  So this message will appear for the normal
optimisation of m0 and the Monte Carlo simulations, but is harmless.
Does this message appear for other model-free models?

Cheers,

Edward


On Tue, Apr 8, 2008 at 4:05 AM, Sébastien Morin
&lt;NO-REPLY.INVALID-ADDRESS@xxxxxxx&gt; wrote:
</pre><blockquote class="blockquote"><pre style="margin: 0em;">

 Follow-up Comment #2, bug #11454 (project relax):

 Hi,

 I had a look at the log files for other runs and found out that the same
 error message appeared for minimization of model &quot;m0&quot; for every diffusion
 tensor. For example, here's what's found for the prolate diffusion tensor
 right after initialization (folder &quot;init&quot;) :


 ============================
 Method of Multipliers
 ~~~~~~~~~~~~~~~~~~~~~
 The constraints have been incorrectly supplied.

 relax&gt; results.write(run='m0', file='results', dir='prolate/round_1/m0',
 force=1, format='columnar', compress_type=1)
 Opening the file 'prolate/round_1/m0/results.bz2' for writing.
 ============================


 The error message appears for every round when at model &quot;m0&quot;.


 For runs without model &quot;m0&quot;, the error message never arises...




 Séb

    _______________________________________________________

 Reply to this item at:

  &lt;<a  rel="nofollow" href="http://gna.org/bugs/?11454">http://gna.org/bugs/?11454</a>&gt;

 _______________________________________________
  Message sent via/by Gna!
  <a  rel="nofollow" href="http://gna.org/">http://gna.org/</a>


</pre></blockquote><pre style="margin: 0em;">


</pre>
<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
</div><!-- end msgdata -->
<br />
<h3><a name="related" href="#related">Related Messages</a></h3>
<div class="relateddata">
<ul><li><strong>Follow-Ups</strong>:
<ul>
<li><strong><a name="00043" href="msg00043.html">Re: [bug #11454] Constraint incorrectly supplied when using the	&quot;full_analysis.py&quot; script</a></strong>
<ul><li><em>From:</em> Sébastien Morin</li></ul></li>
</ul></li></ul>
<!--X-Follow-Ups-End-->
<!--X-References-->
<ul><li><strong>References</strong>:
<ul>
<li><strong><a name="00038" href="msg00038.html">[bug #11454] Constraint incorrectly supplied when using the	&quot;full_analysis.py&quot; script</a></strong>
<ul><li><em>From:</em> Sébastien Morin</li></ul></li>
<li><strong><a name="00039" href="msg00039.html">[bug #11454] Constraint incorrectly supplied when using the	&quot;full_analysis.py&quot; script</a></strong>
<ul><li><em>From:</em> Sébastien Morin</li></ul></li>
<li><strong><a name="00040" href="msg00040.html">[bug #11454] Constraint incorrectly supplied when using the	&quot;full_analysis.py&quot; script</a></strong>
<ul><li><em>From:</em> anonymous</li></ul></li>
<li><strong><a name="00041" href="msg00041.html">[bug #11454] Constraint incorrectly supplied when using the	&quot;full_analysis.py&quot; script</a></strong>
<ul><li><em>From:</em> Sébastien Morin</li></ul></li>
</ul></li></ul>
<!--X-References-End-->
<!--X-BotPNI-->
</div><!-- end relateddata -->
<!-- NoBotLinksApartFromRelatedMessages -->

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
<div class="footer"></div><br />
<div class="right">Powered by <a href="http://www.mhonarc.org">MHonArc</a>, Updated Tue Apr 08 13:41:30 2008</div>  
</body>
</html>
